### PR TITLE
feat(ui): Allow apps to register a custom seek bar UI implementation ( Take 2 )

### PR DESCRIPTION
**Pull request by hichem-code-it**
Hope it will be fine this time !

### CODE REVIEW

**Comment by hichem-code-it** [comment]
@googlebot I consent  ?

**Comment by hichem-code-it** [comment]
@joeyparrish I have no idea why the googlebot still complains, any idea ?

**Comment by TheModMaker** [comment]
The email address in the commits doesn't match the CLA you signed.  Either go back to the [CLA page](https://cla.developers.google.com/clas) and add that email, or change the email in the commits.  You should also check the email you used is attached to your GitHub account.

**Comment by hichem-code-it** [comment]
@googlebot I consent  , with the update github account email !

**Comment by hichem-code-it** [comment]
@TheModMaker The email of the github account was the problem ! Thks for your help !

File: ui/externs/ui.js [hunk 1]
```diff
@@ -357,3 +356,43 @@ shaka.extern.IUISettingsMenu = class {
     this.backSpan;
   }
 };
+
+/**
+ * Interface for SeekBars. SeekBars should inherit from the concrete base
+ * class shaka.ui.Element.
```
Comment (joeyparrish) [inline ui/externs/ui.js:362]: The default inherits from shaka.ui.RangeElement, which manages some tricky details for various older browsers.  Though you could just as well inherit from shaka.ui.Element, too, if you implement the full interface.  Or you could conceivably inherit from shaka.ui.SeekBar.

I guess the imperative is shaka.ui.Element, so you're not wrong.  But we should probably mention also that you can inherit from shaka.ui.RangeElement or shaka.ui.SeekBar, as well, to save yourself some effort.

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Looks great!  Thanks!

File: ui/externs/ui.js (hunk 1 above)
Comment (hichem-code-it) [reply ui/externs/ui.js]: I updated that comment line accordingly !

**Review by joeyparrish — APPROVED** [review]
Thanks!

**Pull request merged**